README.md

```diff
@@ -1,2 +1,71 @@
 # sml-python-method-template
-This repository templates the development of a new python method for the statistical method library
+This repository templates the development of a new python method for the Statistical Method Library
+
+## Table of Contents
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Getting Started
+
+This template is an example repository for a new Python implementation of a method targeted for the Statistical Method Library.
+You should use the template to produce the basic repository structure and tooling useful for developing a method in Python.
+
```

Comment:
Is it worth mentioning where this button can be found?

Reply:
I have tried to add a description that is accessible